suitcase/utils/__init__.py

```diff
@@ -241,6 +234,7 @@ def open(self, label, postfix, mode, encoding=None, errors=None):
         self.buffers[postfix] = buffer
         return buffer
 
+
```

Comment:
Also -- a small thing -- flake8 will complain about this line. :-D

Reply:
sorry, forgot to pass it through flake8 before commiting, I have resolved all the flake8 issues in the latest commit